lib/rubocop/cop/rails/validation.rb

```diff
@@ -48,6 +48,24 @@ def on_send(node)
         def preferred_method(method)
           WHITELIST[BLACKLIST.index(method.to_sym)]
         end
+
+        def autocorrect(node)
+          _receiver, method_name, *args = *node
+          options = args.delete_if { |arg| arg.type == :sym }.first
+          lambda do |corrector|
+            validate_parameter = method_name.to_s.split('_')[1]
+            corrector.replace(node.loc.selector, 'validates')
+            if options
+              corrector.replace(
+                options.loc.expression,
+                "#{validate_parameter}: { #{options.source} }")
```

Comment:
@Neodelf Yes, use `autocorrect_source` in the specs.
@bbatsov In general I think it's fine to let auto-correct produce code according to default configuration rather than checking other cops' configuration. Being "super smart" leads to complex and tightly coupled code IMO. Let the other cops correct it afterwards if the user has configured something non-default.